### ISSUE TITLE
Will the package be updated ?

### ISSUE THREAD

**Issue by sanyoll** [hjae78/Gregicality-Community-Pack#79]
There are a lot of missing items in the mission book, and although I'm only in the low voltage phase now, I'm worried if I can experience the entire game process
I am not a native English speaker and I apologize for the poor presentation of the translation software

**Comment by CrossElectronics** [comment]
From the commit history, I can say that the pack is under active development. So basically you are playing early access. I'm not a dev though, so I cannot guarantee you anything, but the pack is definitely worth trying.

**Comment by htmlcsjs** [comment]
Yes, the pack will be updated, currently most Devs are taking a break to work on CEu

**Comment by sanyoll** [comment]
Thank you for your reply, I'll play it as long as I can
